### DISCUSSION TITLE
Text editing in "in-place" editor broken

### DISCUSSION THREAD

**Discussion by khemarato** [community/community#56481] [Mobile]
### Select Topic Area

Bug

### Body

Hi, not sure if this is related to the new code viewer UI but now when I try to edit a file on mobile using the in-browser "in place" editor, I cannot select text and replace it.

**Comment by eliperkins** [comment]
Hey @buddhist-uni, thanks for your feedback! To best help you out, can I ask you a question? Are you seeing this issue within a web browser on GitHub.com or within the native GitHub Mobile app on iOS or Android?

**Reply by khemarato** [reply]
The Chrome browser on Android
